packages/shared/src/forms/datePicker/DatePicker.ios.js

```diff
@@ -22,7 +26,7 @@ export default class DatePicker extends React.Component<Props, State> {
   static getDerivedStateFromProps = ({ date }: Props) => ({ date });
 
   componentDidMount = () => {
-    this.setState({ date: this.props.date });
+    this.setState({ date: this.props.date || DateUtils.getUTCToday() });
```

Comment:
Why not set `DateUtils.getUTCToday()` as default value for prop `date`?

Reply:
Because if it is called like `<DatePicker date={null} />` it has a value null, and default value just takes place of the prop is missing (`<DatePicker />`).

The the app crashes when you open the datepicker and date = null.

Reply:
Didn't think about that. Makes sense!